proposals/description/tip_111.ts

```diff
@@ -0,0 +1,123 @@
+      arguments: (addresses) => [
+        addresses.ethLidoPCVDepositOld, // pcvDeposit
+        addresses.steth, // token
+        addresses.ethLidoPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Migrate all stETH to new PCVDeposit'
+    },
+    {
+      target: 'balancerGaugeStaker',
+      values: '0',
+      method: 'unstakeFromGauge(address,uint256)',
+      arguments: (addresses) => [
+        addresses.bpt30Fei70Weth, // token
+        '252865858892972812879565' // amount
+      ],
+      description: 'Unstake all B-70WETH-30FEI from Balancer gauge'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.balancerGaugeStaker, // pcvDeposit
+        addresses.bpt30Fei70Weth, // token
+        addresses.balancerDepositFeiWeth, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Move all B-70WETH-30FEI to Balancer deposit'
+    },
+    {
+      target: 'balancerDepositFeiWeth',
+      values: '0',
+      method: 'exitPool(address)',
+      arguments: (addresses) => [addresses.balancerDepositFeiWeth],
+      description: 'Exit LP position and send WETH to self, while burning the FEI'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.balancerDepositFeiWeth, // pcvDeposit
+        addresses.weth, // token
+        addresses.aaveEthPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Move all WETH to Aave PCVDeposit'
+    },
+    {
+      target: 'aaveEthPCVDeposit',
+      values: '0',
+      method: 'deposit()',
+      arguments: () => [],
+      description: 'Deposit all WETH in Aave'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'swapDeposit(address,address)',
+      arguments: (addresses) => [addresses.ethLidoPCVDepositOldWrapper, addresses.ethLidoPCVDeposit],
+      description: 'Swap old & new Lido ETH PCVDeposits in CR Oracle'
+    },
+    {
+      target: 'dpiToDaiLBPSwapper',
+      values: '0',
+      method: 'exitPool(address)',
+      arguments: (addresses) => [addresses.compoundDaiPCVDeposit],
+      description: 'Withdraw all DAI and DPI from LBP pool to the compoundDAIPCVDeposit (~$3.8m)'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.compoundDaiPCVDeposit, // pcvDeposit
+        addresses.dpi, // token
+        addresses.tribalCouncilSafe, // to
+        '10000' // basisPoints, 100%
+      ],
+      description:
+        'Withdraw all DPI from the Compound DAI PCV deposit (~$200k) to the TribalCouncil multisig, where it will be liquidated'
```

Comment:
This is what I had originally put together - send the leftover DPI to the TC safe where it can be liquidated on a DEX aggregator. Flagging here for others, it's a small amount, appropriate imo